### DISCUSSION TITLE
FaceId is released for SDXL

### DISCUSSION THREAD

**Discussion by ut1kaa** [lllyasviel/Fooocus#1819] [Ideas]
Recently released an FaceID SDXL model, it would be cool to see it in fooocus. 
https://huggingface.co/h94/IP-Adapter-FaceID/tree/main

**Comment by yelagin32** [comment]
It works on A1111! Great feature to install to Fooocus too.

**Comment by lslsl3q** [comment]
agreed

**Comment by mashb1t** [comment]
see https://github.com/lllyasviel/Fooocus/issues/1522, it's in the backlog
